core/src/tile/tileManager.h

```diff
@@ -32,11 +32,7 @@ class TileManager {
 
 public:
 
-    /* Returns the single instance of the TileManager */
-    static std::unique_ptr<TileManager> GetInstance() {
```

Comment:
You know, in retrospect I don't think we need to jump through hoops to get the compiler to enforce that `TileManager` is a singleton :)